Learning/collect.py

```diff
@@ -1,6 +1,41 @@
 import subprocess
 import argparse
 import os
+from simulator.sim_globals import NUM_PLANTS, PERCENT_NON_PLANT_CENTERS
+from simulator.plant_type import PlantType
+import simulator.baselines.analytic_policy as analytic_policy
+import multiprocessing as mp
+from data_collection import DataCollection
+
+def start_proc(dir_seed):
+    print(dir_seed[1])
+    rows = 150
+    cols = 150
+    num_plant_types = PlantType().num_plant_types
+    depth = num_plant_types + 3  # +1 for 'earth' type, +1 for water, +1 for health
+    sector_rows = 15
+    sector_cols = 30
+    prune_window_rows = 5
+    prune_window_cols = 5
+    garden_step = 1
```

Comment:
Create vars in sim_globals for SECTOR_ROWS, SECTOR_COLS, PRUNE_WINDOW_ROWS, PRUNE_WINDOW_COLS.

Use STEP from sim_globals.  (the STEP variable was just added to master).